### PR TITLE
Add content about recurring corrections

### PR DESCRIPTION
**Pull request by markazerdd**
### What does this PR do?

Updates to the Status Corrections section of the SLO docs to include new information about recurring corrections and some further clarifications

### Motivation

Status Corrections is imminently headed to GA release out of public beta

### Preview

https://docs-staging.datadoghq.com/markazerdd/slo-corrections-ga/monitors/service_level_objectives/#slo-status-corrections

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/service_level_objectives/_index.md [hunk 1]
```diff
@@ -173,17 +173,32 @@ When you create a correction window for an SLO, the time period you specify is r
 - For monitor-based SLOs, time in the correction window is not counted.
 - For metric-based SLOs, all good and bad events in the correction window are not counted.
 
-Status corrections are in public beta through the [SLO status corrections API][12], [Terraform][13], and the UI.
+You have the option to create one-time corrections for ad-hoc adjustments, or recurring corrections for predictable adjustments that occur on a regular cadence. One-time corrections require a start and end time, while recurring corrections require a start time, duration, and interval. Recurring corrections are based on [iCalender RFC 5545's RRULE specification][14]. Specifying an end time for recurring corrections is optional in case you need the correction to repeat indefinitely. 
+
+For either type of correction, you must select a correction category that states why the correction is being made. The available categories are `Scheduled Maintenance`, `Outside Business Hours`, `Deployment`, `Other`. You can optionally include a description to provide additional context if necessary.
```
Comment (urseberry) [inline content/en/monitors/service_level_objectives/_index.md:178]:
```suggestion
For either type of correction, you must select a correction category that states why the correction is being made. The available categories are `Scheduled Maintenance`, `Outside Business Hours`, `Deployment`, and `Other`. You can optionally include a description to provide additional context if necessary.
```

File: content/en/monitors/service_level_objectives/_index.md [hunk 2]
```diff
@@ -173,17 +173,32 @@ When you create a correction window for an SLO, the time period you specify is r
...
+Each SLO has a maximum limit of corrections that can be configured to ensure query performance. The limits per SLO are as follows:
+
+| Correction Type   | Limit per SLO |
+| ----------------- | ------------- |
+| One-time          | 30            |
+| Daily recurring   | 2             |
+| Weekly recurring  | 3             |
+| Monthly recurring | 5             |
+
+Status corrections are configurable through the UI, the [SLO status corrections API][12], or a [Terraform resource][13].
```
Comment (urseberry) [inline content/en/monitors/service_level_objectives/_index.md:189]:
```suggestion
You may configure status corrections through the UI, the [SLO status corrections API][12], or a [Terraform resource][13].
```

File: content/en/monitors/service_level_objectives/_index.md [hunk 3]
```diff
@@ -173,17 +173,32 @@ When you create a correction window for an SLO, the time period you specify is r
...
 
 To access SLO status corrections in the UI:
 
 1. Create a new SLO or click on an existing one.
-2. Navigate to an SLO’s detailed side panel view.
+2. Navigate to an SLO’s details side panel view.
 3. Under the gear icon, select **Correct Status** to access the **Status Corrections** creation modal.
-4. Select the **Time Correction Window**, **Correction Type**, and add **Notes**.
-5. Click **Apply Correction**.
+4. Choose between `One-Time` and `Recurring` in the **Select the Time Correction Window**, and specify the time period you which to correct.
```
Comment (urseberry) [inline content/en/monitors/service_level_objectives/_index.md:198]:
```suggestion
4. Choose between `One-Time` and `Recurring` in the **Select the Time Correction Window**, and specify the time period you wish to correct.
```

**Review by urseberry — COMMENTED** [review]
Hey, you wrote these updates very nicely! I do have some small corrections.

**Review by iminoso — APPROVED** [review]
lgtm for slo app side

**Review by urseberry — APPROVED** [review]

**Comment by urseberry** [comment]
Mark Azer will tell the docs team when this is safe to merge. They need to turn on some feature flags before we release the documentation.

**Pull request merged**